app/services/hyrax/collection_types/create_service.rb

```diff
@@ -133,7 +133,7 @@ def self.add_participants(collection_type_id, participants)
             agent_id = p.fetch(:agent_id)
             access = p.fetch(:access)
             Hyrax::CollectionTypeParticipant.create!(hyrax_collection_type_id: collection_type_id, agent_type: agent_type, agent_id: agent_id, access: access)
-          rescue => e # rubocop:disable Lint/RescueWithoutErrorClass
+          rescue => e
```

Comment:
I really wish we called this out as something that should be fixed.  Can we add this cop to Bixby?

Reply:
This cop is new since bixby launched. I've opened a [bixby ticket](https://github.com/samvera-labs/bixby/issues/23), but it will be a 2.0 feature.

In the meanwhile, we can add this cop locally. However, I'm not sure what the value of doing that is at the moment, if we are just going to disable it in the individual cases.

Reply:
@jcoyne This cop has also been replaced with `Style/RescueStandardError`.

Honestly, I'm looking at this now and pretty unconvinced that this is a Good Cop :tm:. In this case, we're catching `StandardError` and logging, which seems like an extremely common and happy use case.

Reply:
I disagree that this is a happy use case. Currently I have no idea what error we're expecting here.  Could it be a KeyError or a Database error (out of space maybe?) I'd really like for those exceptions to be bubbling up.

Reply:
I've added https://github.com/samvera/hyrax/pull/2666 to address this case in particular, but I'd really think that over general error catching is likely a code smell.

Reply:
I opened a Bixby issue at https://github.com/samvera-labs/bixby/issues/23; but resolving it will be a major `bixby` version. I'd suggest we discuss there for a bit.

If we want to add this locally in the meanwhile, I can do that. I'd prefer not to block this PR for it, since the only current instance of the issue is now fixed.

Reply:
catching StandardError is IMO fine to do when you are not actually swallowing the error but: a) logging and re-raising original, and/or b) re-raising another module-specific error wrapping the original. 

I do both frequently. It would be annoying if I had to tell rubocop to let me do it all the time, but then, that may just be the way it is with rubocop.